wagn/lib/wagn/mods_spec_helper.rb

```diff
@@ -1,6 +1,5 @@
 # -*- encoding : utf-8 -*-
 require 'spork'
-raise "is this loaded anywhere?"
```

Comment:
The place where this is required.  I think it may have gotten forked into a file in card and wagn gems.  Probably we don't need two, except for some legacy cases maybe?  Or is it really needed as a distinct thing for mod-dev decks.  Maybe the template would change not to need it.